assets/src/dashboard/components/pageStructure/index.js

```diff
@@ -179,9 +180,10 @@ export function LeftRail() {
     >
       <div ref={upperContentRef}>
         <Content>
-          <WebStoriesHeading>
+          <WebStoriesLogo title={__('Web Stories', 'web-stories')} />
+          <VisuallyHiddenHeading>
             {__('Web Stories', 'web-stories')}
-          </WebStoriesHeading>
+          </VisuallyHiddenHeading>
```

Comment:
I don't think this is needed.

With the latest change, `<WebStoriesLogo title={__('Web Stories', 'web-stories')} />` will result in the SVG containing a `<title>` child, which is what screen readers will  see (basically like the `alt` attribute on images.

If you keep `<VisuallyHiddenHeading>`, you'll hear `Web Stories` twice.

Reply:
It does read it twice. I say once we drop one method then this is good to go!

Reply:
Updated this to just use the title. I still think that we need an h1 tag for html hierarchical purposes within the dashboard but I know that the screen reader here is the priority so I have removed the h1 tag.

Reply:
Just wrap the svg in the h1?

Reply:
Let's leave this be for now, there's design changes to the left nav coming down the pipeline and we can address it better there.